src/components/CountriesList/CountriesList.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+import './CountriesList.css'
+import countriesFrom from '../../countries.json'
+
+class CountriesList extends Component {
+    constructor() {
+        super()
+        this.state = {
+            countries: countriesFrom
+        }
+    }
+    render() {
+        return (
+            <>
+            <div className="col-5 list">
+                <div className="list-group">
+                    {this.state.countries.map(elm => <NavLink activeStyle={{ fontWeight: "800" }} key={elm.css3} className="list-group-item list-group-item-action" to={`/country/${elm.cca3}`}><span>{elm.flag}</span>{elm.name.common}</NavLink>)}
```

Comment:
```suggestion
                    {this.state.countries.map((elm, idx) => <NavLink activeStyle={{ fontWeight: "800" }} key={idx} css3={elm.css3} className="list-group-item list-group-item-action" to={`/country/${elm.cca3}`}><span>{elm.flag}</span>{elm.name.common}</NavLink>)}
```

Las keys ayudan a React a identificar que ítems han cambiado, son agregados, o son eliminados. Las keys deben ser dadas a los elementos dentro del array para darle a los elementos una identidad estable.

Cuando no tengas IDs estables para renderizar, puedes usar el índice del ítem como una key como último recurso.

No es recomendable usar índices para keys si el orden de los ítems puede cambiar. Esto puede impactar negativamente el rendimiento y puede causar problemas con el estado del componente.